src/middlewared/middlewared/plugins/sysdataset.py

```diff
@@ -113,13 +113,28 @@ class Config:
                     f'Pool "{new["pool"]}" has an encryption passphrase set. '
                     'The system dataset cannot be placed on this pool.'
                 )
+            elif await self.middleware.call(
+                'pool.dataset.query', [
+                    ['name', '=', new['pool']], ['encrypted', '=', True],
+                    ['OR', [['key_format.value', '=', 'PASSPHRASE'], ['locked', '=', True]]]
+                ]
+            ):
+                verrors.add(
+                    'sysdataset_update.pool',
+                    'The system dataset cannot be placed on a pool '
+                    'which has root dataset encrypted with PASSPHRASE or is locked.'
```

Comment:
Capitalization is again confusing to the user.  It seems to mean the PASSPHRASE method, not just a passphrase.
```suggestion
                    'which has the root dataset encrypted with PASSPHRASE or locked.'
```

Reply:
I am not sure i follow "PASSPHRASE method" ? There are 3 types of keys which can be used to encrypt a dataset and passphrase is one of them. So if a dataset is encrypted with passphrase key type, we don't want to allow system dataset to be placed there of if course the root dataset is locked as well. I don't think `or locked` is okay in this context, please advise. Thank you

Reply:
As above, this mixes Python capitalization with spoken (read) words. When speaking to the end user about a key, use "a raw key, a hex key, or a passphrase key".  If you are speaking to the user about Python constants/methods/other code, then use the uppercase names.